src/chrome/content/rules/NVidia.xml

```diff
@@ -136,72 +79,19 @@
 	<target host="ngvpn35.nvidia.com" />
 	<target host="ngvpn36.nvidia.com" />
 	<target host="ngvpn37.nvidia.com" />
+	<target host="omniture.nvidia.com" />
 	<target host="partners.nvbugs.nvidia.com" />
 	<target host="nvdeveloper.nvidia.com" />
-	<!--target host="nvidianews.nvidia.com" /-->
 	<target host="partners.nvidia.com" />
 	<target host="es.pforce.nvidia.com" />
 	<target host="redeem.nvidia.com" />
-	<!--target host="registration.nvidia.com" /-->
 	<target host="shield.nvidia.com" />
-	<target host="store.nvidia.com" />
```

Comment:
Why did you remove this host?

Reply:
Probably because it's broken in some way.

Reply:
I can't see any breakage now, reenabling.